src/Features/CSharp/Portable/ConvertToRawString/ConvertRegularStringToRawStringCodeRefactoringProvider.cs

```diff
@@ -37,12 +41,25 @@ private enum ConvertToRawKind
             MultiLineWithoutLeadingWhitespace,
         }
 
+        private static readonly BidirectionalMap<ConvertToRawKind, string> s_kindToEquivalenceKeyMap =
```

Comment:
Reverse direction is needed when computing relevant fixes for a given equivalence key during FixAll